java/client/src/test/java/glide/connectors/handlers/MessageHandlerTests.java

```diff
@@ -0,0 +1,226 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.connectors.handlers;
+
+import static glide.api.models.GlideString.gs;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import glide.api.models.PubSubMessage;
+import glide.api.models.configuration.BaseSubscriptionConfiguration;
+import glide.api.models.exceptions.RedisException;
+import glide.managers.BaseResponseResolver;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import response.ResponseOuterClass;
+
+/** Unit tests for MessageHandler */
+public class MessageHandlerTests {
+    @Test
+    public void testExactMessage() throws Exception {
```

Comment:
You can rename all tests names into sneak_case, we have a fixture witch replaces `_` by ` ` in the test report

Reply:
Done.